README.md

```diff
@@ -6,8 +6,13 @@ This repository contains IaC code based on Pulumi to provision dynamic test infr
 
 To run scripts and code in this repository, you will need:
 
- * [Go](https://golang.org/doc/install) 1.19 or later. You'll also need to set your `$GOPATH` and have `$GOPATH/bin` in your path.
- * Python 3.7+ along with development libraries for tooling.
+* [Go](https://golang.org/doc/install) 1.19 or later. You'll also need to set your `$GOPATH` and have `$GOPATH/bin` in your path.
+* Python 3.7+ along with development libraries for tooling.
+* `account-admin` role on AWS `agent-sandbox` account. Ensure it by running
+  
+  ```bash
+  aws-vault login agent-sandbox-account-admin
```

Comment:
Recommended to use the SSO role: `sso-agent-sandbox-account-admin`

Reply:
Ah, interesting ! Thank you

Reply:
I tried using this role, but I get an error:

```bash
aws-vault login sso-agent-sandbox-account-admin                                                                                                        
aws-vault: error: login: profile sso-agent-sandbox-account-admin: No master credentials found
```

Using `agent-sandbox-account-admin` works fine. Anything I should check ?

Reply:
I think it’s because you should re-generate your `~/.aws/config` with this: https://github.com/DataDog/cloud-inventory/blob/master/organizations/aws/sso/README.md#aws-cli-config-setup--update in order to get the new SSO profiles.

Reply:
Thank you ! It worked perfectly !